iati/core/tests/test_resources.py

```diff
@@ -113,6 +113,12 @@ def test_find_schema_paths_file_extension(self, get_schema_path_function):
         for path in paths:
             assert path[-4:] == iati.core.resources.FILE_SCHEMA_EXTENSION
 
+    def test_get_test_data_paths_in_folder(self):
+        """Check that test data is being found in specified subfolders."""
+        paths = iati.core.resources.get_test_data_paths_in_folder('ssot-activity-xml-fail')
+
+        assert len(paths) == 237
```

Comment:
Wondering if some of these 'number of expected files' should be consolidated somewhere, rather than defined as magic numbers - perhaps in `iati/core/tests/utilities.py`, or elsewhere, even a new file: `iati/core/tests/constants.py` ??

Reply:
Once multi-version support is added, probably https://github.com/IATI/iati.core/pull/142/commits/7bc397fd2303f6b87a9c486178a8ede36891b7f8